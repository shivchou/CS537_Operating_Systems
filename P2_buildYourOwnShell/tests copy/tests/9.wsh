alias e = echo
which e
which alias
which which
which path
which cd
which history
which exit
which unalias
which echo
which cat
which /bin/echo
which ./notarealfile
which notacommand
which
which echo extra
alias e = exit
which e
path tmp
which echo
which cat
path /bin
which echo
