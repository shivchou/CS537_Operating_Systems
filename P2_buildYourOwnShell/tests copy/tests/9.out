e: aliased to 'echo'
alias: wsh builtin
which: wsh builtin
path: wsh builtin
cd: wsh builtin
history: wsh builtin
exit: wsh builtin
unalias: wsh builtin
echo: found at /bin/echo
cat: found at /bin/cat
/bin/echo: found at /bin/echo
./notarealfile: not found
notacommand: not found
e: aliased to 'exit'
echo: not found
cat: not found
echo: found at /bin/echo
